cmd/soroban-cli/src/commands/events.rs

```diff
@@ -226,11 +218,22 @@ impl NetworkRunnable for Cmd {
         client
             .verify_network_passphrase(Some(&network.network_passphrase))
             .await?;
+
+        let contract_ids: Vec<String> = self
+            .contract_ids
+            .iter()
+            .map(|id| {
+                self.locator
+                    .resolve_contract_id(id, &network.network_passphrase)
+                    .map(|contract| contract.to_string())
```

Comment:
```suggestion
                Ok(self.locator
                    .resolve_contract_id(id, &network.network_passphrase)?.to_string())
```
I'm fairly confident this should work.

Reply:
Almost:

```diff
diff --git a/cmd/soroban-cli/src/commands/events.rs b/cmd/soroban-cli/src/commands/events.rs
index 2eefb02..139fdc0 100644
--- a/cmd/soroban-cli/src/commands/events.rs
+++ b/cmd/soroban-cli/src/commands/events.rs
@@ -223,9 +223,11 @@ impl NetworkRunnable for Cmd {
             .contract_ids
             .iter()
             .map(|id| {
-                self.locator
-                    .resolve_contract_id(id, &network.network_passphrase)
-                    .map(|contract| contract.to_string())
+                Ok::<String, Error>(
+                    self.locator
+                        .resolve_contract_id(id, &network.network_passphrase)?
+                        .to_string(),
+                )
             })
             .collect::<Result<Vec<_>, _>>()?;
```